### ISSUE TITLE
Bug: Target words are duplicated in wA 

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#6405]
Robert reported this bug using https://tc-builds.door43.org/build/release-v2.0.0/translationCore-linux-x64-2.0.0-ade72a0.deb (Linux build)

![image.png](https://images.zenhubusercontent.com/55e5f3984f7a08fbb7f4a658/e7881eab-7320-4962-9a64-f327edb07b56)

"Look at the last two sets of words. My two last steps were: 1/ drag 'about' into 'thirty', 2/ combine the last two Heb words."

Project: 
[en_ust_jdg_book.zip](https://github.com/unfoldingWord-dev/translationCore/files/3533885/en_ust_jdg_book.zip)

Log:
[Thu Aug 22 2019.log](https://github.com/unfoldingWord-dev/translationCore/files/3533894/Thu.Aug.22.2019.log)

**Comment by deferredreward** [comment]
This also occasionally happens after editing the target text. I'll update attachments if I encounter again.

**Comment by cckozie** [comment]
This is still a problem in 2.1.0 (46a3795)
Import this file: 
[07-JDG.usfm.zip](https://github.com/unfoldingWord/translationCore/files/4026873/07-JDG.usfm.zip)
Load the project in wA
Navigate to verse 20:31
Drag "about" into the tile containing "thirty"
Combine the last two Hebrew words

**Comment by da1nerd** [comment]
Do those words also get duplicated in the wordbank on the left?

**Comment by deferredreward** [comment]
Not when I was observing it, that was before the latest update.

**Comment by cckozie** [comment]
@neutrinog No, the words are not duplicated in the word bank, just in the alignment area.

I am unable to duplicate this problem in 2.1.0 (e61795c). There have been many changes to the hbo which cause changes to suggestions. There may be other factors involved as well.
It can still be duplicated using the build, project, and steps in my previous comment.

**Comment by cckozie** [comment]
Unable to duplicate in 2.2.0 (2375ae5), but am leaving it open in case the problem is reported again.

**Comment by birchamp** [comment]
@cckozie can we put this is QA watch or close it?

**Comment by cckozie** [comment]
@birchamp - Close it if you'd like. QA Watch List is primarily for watching for things that we think may be related to issues being worked on.

**Comment by cckozie** [comment]
The problem is back in 3.0.0 (74b79f2) 
![image](https://user-images.githubusercontent.com/8171759/88845261-82103580-d1a9-11ea-9368-39438056e4fa.png)

**Comment by deferredreward** [comment]
![image](https://user-images.githubusercontent.com/703973/90558533-3a5a4980-e16a-11ea-9a2e-bb69114c815c.png)
Edited target text.
